build/FindQt5Transitional.cmake

```diff
@@ -23,8 +23,8 @@ if (Qt5Core_FOUND)
         OpenGL
         LinguistTools
         Help
-        WebEngine
-        WebEngineWidgets
+#        WebEngine
+#        WebEngineWidgets
```

Comment:
Thwere got to be, as this way it doesn't work. The cmake warningis are annoying though